// @expo-google-fonts/open-sans/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const OpenSans_300Light = require('./300Light/OpenSans_300Light.ttf');
export const OpenSans_400Regular = require('./400Regular/OpenSans_400Regular.ttf');
export const OpenSans_500Medium = require('./500Medium/OpenSans_500Medium.ttf');
export const OpenSans_600SemiBold = require('./600SemiBold/OpenSans_600SemiBold.ttf');
export const OpenSans_700Bold = require('./700Bold/OpenSans_700Bold.ttf');
export const OpenSans_800ExtraBold = require('./800ExtraBold/OpenSans_800ExtraBold.ttf');
export const OpenSans_300Light_Italic = require('./300Light_Italic/OpenSans_300Light_Italic.ttf');
export const OpenSans_400Regular_Italic = require('./400Regular_Italic/OpenSans_400Regular_Italic.ttf');
export const OpenSans_500Medium_Italic = require('./500Medium_Italic/OpenSans_500Medium_Italic.ttf');
export const OpenSans_600SemiBold_Italic = require('./600SemiBold_Italic/OpenSans_600SemiBold_Italic.ttf');
export const OpenSans_700Bold_Italic = require('./700Bold_Italic/OpenSans_700Bold_Italic.ttf');
export const OpenSans_800ExtraBold_Italic = require('./800ExtraBold_Italic/OpenSans_800ExtraBold_Italic.ttf');
