// @expo-google-fonts/raleway/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Raleway_100Thin = require('./100Thin/Raleway_100Thin.ttf');
export const Raleway_200ExtraLight = require('./200ExtraLight/Raleway_200ExtraLight.ttf');
export const Raleway_300Light = require('./300Light/Raleway_300Light.ttf');
export const Raleway_400Regular = require('./400Regular/Raleway_400Regular.ttf');
export const Raleway_500Medium = require('./500Medium/Raleway_500Medium.ttf');
export const Raleway_600SemiBold = require('./600SemiBold/Raleway_600SemiBold.ttf');
export const Raleway_700Bold = require('./700Bold/Raleway_700Bold.ttf');
export const Raleway_800ExtraBold = require('./800ExtraBold/Raleway_800ExtraBold.ttf');
export const Raleway_900Black = require('./900Black/Raleway_900Black.ttf');
export const Raleway_100Thin_Italic = require('./100Thin_Italic/Raleway_100Thin_Italic.ttf');
export const Raleway_200ExtraLight_Italic = require('./200ExtraLight_Italic/Raleway_200ExtraLight_Italic.ttf');
export const Raleway_300Light_Italic = require('./300Light_Italic/Raleway_300Light_Italic.ttf');
export const Raleway_400Regular_Italic = require('./400Regular_Italic/Raleway_400Regular_Italic.ttf');
export const Raleway_500Medium_Italic = require('./500Medium_Italic/Raleway_500Medium_Italic.ttf');
export const Raleway_600SemiBold_Italic = require('./600SemiBold_Italic/Raleway_600SemiBold_Italic.ttf');
export const Raleway_700Bold_Italic = require('./700Bold_Italic/Raleway_700Bold_Italic.ttf');
export const Raleway_800ExtraBold_Italic = require('./800ExtraBold_Italic/Raleway_800ExtraBold_Italic.ttf');
export const Raleway_900Black_Italic = require('./900Black_Italic/Raleway_900Black_Italic.ttf');
