// @expo-google-fonts/roboto/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Roboto_100Thin = require('./100Thin/Roboto_100Thin.ttf');
export const Roboto_200ExtraLight = require('./200ExtraLight/Roboto_200ExtraLight.ttf');
export const Roboto_300Light = require('./300Light/Roboto_300Light.ttf');
export const Roboto_400Regular = require('./400Regular/Roboto_400Regular.ttf');
export const Roboto_500Medium = require('./500Medium/Roboto_500Medium.ttf');
export const Roboto_600SemiBold = require('./600SemiBold/Roboto_600SemiBold.ttf');
export const Roboto_700Bold = require('./700Bold/Roboto_700Bold.ttf');
export const Roboto_800ExtraBold = require('./800ExtraBold/Roboto_800ExtraBold.ttf');
export const Roboto_900Black = require('./900Black/Roboto_900Black.ttf');
export const Roboto_100Thin_Italic = require('./100Thin_Italic/Roboto_100Thin_Italic.ttf');
export const Roboto_200ExtraLight_Italic = require('./200ExtraLight_Italic/Roboto_200ExtraLight_Italic.ttf');
export const Roboto_300Light_Italic = require('./300Light_Italic/Roboto_300Light_Italic.ttf');
export const Roboto_400Regular_Italic = require('./400Regular_Italic/Roboto_400Regular_Italic.ttf');
export const Roboto_500Medium_Italic = require('./500Medium_Italic/Roboto_500Medium_Italic.ttf');
export const Roboto_600SemiBold_Italic = require('./600SemiBold_Italic/Roboto_600SemiBold_Italic.ttf');
export const Roboto_700Bold_Italic = require('./700Bold_Italic/Roboto_700Bold_Italic.ttf');
export const Roboto_800ExtraBold_Italic = require('./800ExtraBold_Italic/Roboto_800ExtraBold_Italic.ttf');
export const Roboto_900Black_Italic = require('./900Black_Italic/Roboto_900Black_Italic.ttf');
